README.md

```diff
@@ -1,461 +1,74 @@
-resolved template). This final step is needed to error/warn user of remaining drift that validation
-steps may not catch
-
-- E.g use case: reference may have a hardcoded field such as a namespace name and the user must comply.
-
-The primary output of this step is a side-by-side diff as shown in the output section above. To
-achieve this meaningful diff the tool must do perform two operations:
-
-1. Render the CRs into a comparable format. This involves doing a hierarchical sorting of the keys
-   to ensure consistent ordering when the CRs are rendered.
-1. Perform the diff
-
-### Workflow Description
-
-To Compare a known valid reference configuration with a live cluster:
+## Run
 
-`kubectl cluster-compare -r <referenceConfigurationDirectory>`
-
-To Compare a known valid reference configuration with a local set of CRs:
-
-`kubectl cluster-compare -r <referenceConfigurationDirectory> -f <inputConfiguration>`
-
-To Compare a known valid reference configuration with a live cluster and with a user config:
-
-`kubectl cluster-compare -r <referenceConfigurationDirectory> -c <userConfig>`
-
-To Run a known valid reference configuration against a support archive:
-
-`kubectl cluster-compare -r <referenceConfigurationDirectory> -f "must-gather*/*/cluster-scoped-resources","must-gather*/*/namespaces" -R`
-
-#### Reference Configuration Directory
-
-#### metadata.yaml
-
-The metadata.yaml is a mandatory file for each reference config. The commands entrypoint will be looking for the
-metadata.yaml file in the reference directory. The name of the file is fixed and cant be changed.
-
-The main thing included in the metadata are the list of reference CRs that are grouped by components and parts (as
-described in previous sections). The Parts are specified under the Parts key in the YAML and include a list of
-components under the Components key. The full schema can be found in the appendix.
-
-Another parameter that can be set in the metadata.yaml file is the templateFunctionFiles. This Implementation of the
-command supports the declaration of nested templates in external files that then can be used in all resource templates
-included in the reference. All files including nested templates should be added to the list of files under the
-templateFunctionFiles key.
-
-Also the metadata,yaml includes an optional field: `fieldsToOmit`. Under this key they can specify fields that should
-not appear in the commands output. The fields will not be reported showed in the output for all templates in the
-reference, meaning no need to specify them in the resource templates. The fields included will not be showed in the
-output even if they are specified in the resource templates. Omitted fields can be nested therefore each field is
-represented by a list of strings. As can be seen in the example below.
-
-Example for metadata.yaml:
-
-```yaml
-
-Parts:
-  - name: guestbook
```

Comment:
In readme can we have yaml block at shows all the possible metadata.yaml keys with a comment for explanation? And also a directory tree with example references?

Reply:
I've added an explained example, but in the interests of brevity and ease of maintenance I've not included every possible key/value or a directory tree as those are covered elsewhere in the repo.
I opted instead for giving an overview and referring the reader to the reference config guide, and the test data for more information and examples. Does that cover what you wanted to see in the main readme?

Reply:
Sounds good!